src/compiler/types.ts

```diff
@@ -1212,11 +1212,11 @@ namespace ts {
          */
         emit(targetSourceFile?: SourceFile, writeFile?: WriteFileCallback): EmitResult;
 
-        getSyntacticDiagnostics(sourceFile?: SourceFile): Diagnostic[];
+        getOptionsDiagnostics(): Diagnostic[];

```

Comment:
Isn't everything in this API implicitly about the compiler anyways?   Like, we don't have "getCompilerSemanticDiagnostics" :)